packages/graph/README.md

```diff
@@ -7,7 +7,7 @@
 ```shell
 touch .env.local
 docker volume create --name hash-graph-data
-docker-compose -f deployment/docker-compose.yml up
+docker-compose -f deployment/docker-compose.yml --env-file .env up
```

Comment:
Added this so that we can make use of environment variables in the script directly (for example so we can start using the `POSTGRES_PORT` env var)

Reply:
Should this take into account `.env.local` as well?